scripts/github-actions.bash

```diff
@@ -0,0 +1,30 @@
+printf "\e[33;1m%s\e[0m\n" '=== Flutter Formatter ==='
```

Comment:
Can we please move this out of this PR and have a separate one later?

Reply:
yes, removed that file from this PR.